lib/ex_machina.ex

```diff
@@ -4,6 +4,8 @@ defmodule ExMachina do
 
   ## Examples
 
+  Put this somewhere like `test/support/ex_machina.ex`
```

Comment:
`Create a file in your application for ex machina: e.g. `test/support/ex_machina.ex``?